<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_rgba_filter</class>
 <widget class="QDialog" name="window_rgba_filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RGBA-фильтр</string>
  </property>
  <widget class="QRadioButton" name="radioB_Red">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>137</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Красный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioB_Green">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Зеленый</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioB_Blue">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Синий</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioB_Alpha">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Альфа-канал</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Укажите значения канала (0-255):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Укажите какой канал хотите изменить:</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>169</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
